locales/pl_PL/pcsx2_Iconized.po

```diff
@@ -636,30 +666,39 @@ msgid ""
 "games will have higher internal framerates. FMVs may be slow. May cause "
 "stability problems."
 msgstr ""
+"300% szybkości cyklu. Ekstremalny wzrost wymagań procesora. Gry ze zmienną "
+"liczbą klatek na sekundę będą miały wyższe liczby klatek na sekundę. FMV "
+"mogą być wolne. Może powodować problemy ze stabilnością."
 
 #: pcsx2/gui/Panels/SpeedhacksPanel.cpp:81
-#, fuzzy
 msgid "Disables EE Cycle Skipping. Most compatible setting."
 msgstr ""
-"0 - Deaktywuje podkradanie cykli VU. Najbardziej kompatybilne ustawienie!"
+"0 - Deaktywuje podkradanie cykli VU. Najbardziej kompatybilne ustawienie."
```

Comment:
Can't read Polish, but VU probably isn't right when it's referring to EE.

Reply:
Yeah, thanks for catching it. Original translation had a lot of copy-paste stuff, I might miss. Will be keep reviewing it.

Reply:
Fixed! ^-^